embulk-output-sqlserver/src/main/java/org/embulk/output/sqlserver/NativeBatchInsert.java

```diff
@@ -62,8 +66,8 @@ public NativeBatchInsert(String server, int port, Optional<String> instance,
     public void prepare(TableIdentifier loadTable, JdbcSchema insertSchema) throws SQLException
     {
         columnCount = insertSchema.getCount();
-        client.open(server, port, instance, database, user, password, loadTable.getTableName());
-
+        client.open(server, port, instance, database, user, password, loadTable.getTableName(), nativeDriverName);
+        client.setDatabaseCharset(databaseEncoding);
```

Comment:
NativeClientWrapper should not set "MS932" for charset in the constructor because the charset will be always overwritten later. 
https://github.com/kitsuyui/embulk-output-jdbc/blob/71207b74e16be2bea8d3761e2fd11063c8222432/embulk-output-sqlserver/src/main/java/org/embulk/output/sqlserver/nativeclient/NativeClientWrapper.java#L67

Would you change as follows?
- add `databaseEncoding` parameter to NativeClientWrapper#open method
- initialize charset and wideCharset in the NativeClientWrapper#open method, not in the NativeClientWrapper constructor.

Reply:
Okay!